### DISCUSSION TITLE
Crack-on-SEI example results differently from documentation

### DISCUSSION THREAD

**Discussion by lazyjobseeker** [pybamm-team/PyBaMM#4093] [Q&A] [answered]
Hello all, I wish to get any help on the reason why running [crack-on-sei example](https://docs.pybamm.org/en/stable/source/examples/notebooks/models/SEI-on-cracks.html) shows a result different from what the documentation shows.

I downloaded .ipnyb file from above example page and ran it on my local without any change.  It outputs seemingly identical V-t curves for cracking/non-cracking models, which is not a case in the documentation.

Would there be any points I am missing in running the example code?  I would truly appreciate any help.  Thanks!

What I get as output:
![image](https://github.com/pybamm-team/PyBaMM/assets/94539544/566e70f0-1fe1-4a7c-9fca-961c6ff30d2a)

From example documentation:
![image](https://github.com/pybamm-team/PyBaMM/assets/94539544/1b67019c-78de-49f0-87fd-56f148545d3d)

**Comment by valentinsulzer** [comment]
@DrSOKane any ideas what could be happening here?

**Comment by DrSOKane** [comment, marked as answer]
The cause of the voltage curves being different is not the cracking, but the stress-induced diffusion that is enabled by default if the particle mechanics option is set to anything other than "none". I will re-write the notebook so that it compares "swelling only" with "swelling and cracking", which would be a fairer comparison.

**Reply by lazyjobseeker** [reply]
@DrSOKane  Thank you so much for your reply!  Then could I consider the V-t graphs I obtained are correct as you intend, as they are both from particle kinetics other than "none" and it would have enabled stress-induced diffusion physics for both models alike?

**Reply by DrSOKane** [reply]
@lazyjobseeker Correct.

**Comment by valentinsulzer** [comment]
Thanks Simon!
